.github/workflows/book.yml

```diff
@@ -0,0 +1,30 @@
+name: mdbook
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-and-upload-to-s3:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Setup mdBook
+        uses: peaceiris/actions-mdbook@v1
+        with:
+          mdbook-version: '0.3.5'
+
+      - run: mdbook build
+        working-directory: book
+
+      - uses: jakejarvis/s3-sync-action@master
```

Comment:
Are we able to pin this to a specific commit instead of `master`?

Just in case Jake decides to get nefarious :)

Reply:
You never now with Jake!

Reply:
✅